tokens/tokens.md

```diff
@@ -0,0 +1,144 @@
+# Aeternity Native Tokens
+
+## Overview
+
+We need to distinguish between a specification of a token, and a
+countable amount of the token.  In this document we use **ANT**
+(Aeternity Native Token) to represent the specification, and **native
+token** or simply **token** to represent the countable asset.
+
+Informally, when you need a new type of token for whatever purpose,
+you create an ANT describing the token, and then you can mint new
+tokens and start sending them to accounts.
+
+Owners of the tokens can spend them to other accounts, or trade them
+(atomically) for other assets with other accounts. If tokens are no
+longer wanted you can burn them.
+
+Only the owner of the ANT can mint new tokens, and is the owner
```

Comment:
--> "Only the owner of the ANT can mint new tokens. Only the owner
decides the total amount of token. The ANT can be finalized, preventing " ...